package.json

```diff
@@ -55,6 +56,7 @@
     "retry": "0.6.1",
     "strophe": "1.2.4",
     "strophejs-plugins": "0.0.7",
+    "styled-components": "1.3.0",
```

Comment:
For now I've pulled the styled-components changes out into a separate commit and moved it before my changes. I've messaged the maintainer of single-select, asking what the expectation is here. Instead of making a pull request now with the styled-components commit like I said I would, I'm going to hold off until getting an answer.